roles/tf2/files/50unattended-upgrades

```diff
@@ -4,7 +4,7 @@
 // from non-security sources (e.g. chromium). By allowing the release
```

Comment:
Apparently, this (generated?) file should not have been committed in the first place, and even if we did want to use the autoupgrade mechanism, its in the wrong role anyway.